examples/image_classification_tflite/ImageClassificationModel.js

```diff
@@ -20,16 +20,14 @@ class ImageClassificationModel {
         throw Error('Fails to initialize neural network context');
       }
       this._nn = nnNative;
-    } else if (this._backend === 'WASM' || this._backend === 'WebGL2') {
+    } else if (this._backend === 'WASM' || this._backend === 'WebGL') {
       this._nn = nnPolyfill;
     }
   }
 
   async createCompiledModel() {
     let options = {};
-    if (this._backend === 'WebGL2') {
-      options.useWebGL2 = true;
-    }
+    options.backend = this._backend;
     this._model = await this._nn.createModel(options);
```

Comment:
Replaced `options.useWebGL2` with `options.backend`.
For polyfill, `options.backend` should be the string 'WASM' or 'WebGL'.